code/modules/mob/dead/observer/spells.dm

```diff
@@ -100,7 +100,12 @@ var/global/list/boo_phrases_silicon=list(
 /spell/targeted/ghost/toggle_darkness/cast()
 	var/mob/dead/observer/ghost = holder
 	ASSERT(istype(ghost))
-	ghost.toggle_darkness()
+	var/choice = input(ghost, "What level of darkness do you want to see?") as null|anything in list("Full darkness", "See through walls")
```

Comment:
doing this with a choice window instead of just toggling to the next mode might be annoying

Reply:
I'm not sure either, but it's something maybe best left to experiment. I personally don't mind the choice.